app/assets/javascripts/components/ConditionsFormField.jsx

```diff
@@ -99,14 +104,32 @@ class ConditionsFormField extends React.Component {
       label: I18n.t('activerecord.attributes.condition.op'),
       options: this.state.operator_options
     };
+    let destroy_field_props = {
+      type: "hidden",
+      name: `${name_prefix}[_destroy]`,
+      id: `${id_prefix}__destroy`,
+      key: `${id_prefix}__destroy`,
+      value: this.state.destroy ? "1" : "0",
+    }
     let value_field_props = this.buildValueProps(name_prefix, id_prefix);
-    return (
-      <div className="condition-fields">
-        <input {...condition_field_props}/>
-        <FormField {...ref_qing_field_props} />
-        <FormField {...operator_field_props} />
-        <FormField {...value_field_props} />
-      </div>
-    );
+    if(this.state.destroy == true) {
```

Comment:
I think there is functionally no difference between `x == true` and `x` in this context. If you want to match `true` exactly you need `===`